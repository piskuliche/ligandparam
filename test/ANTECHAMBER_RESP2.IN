Resp charges for organic molecule

 &cntrl

 nmol = 1,
 ihfree = 1,
 ioutopt = 1,
 iqopt = 2,
 qwt =  0.00100,

 &end
    1.0
Resp charges for organic molecule
    0   41
    6    0
    6    0
    1    0
    1    3
    1    0
    1    5
    6  -99
    8  -99
    7  -99
    1  -99
    6  -99
    6  -99
    6  -99
    6    0
    6  -99
    6  -99
    1    0
    6    0
    6  -99
    1  -99
    6    0
    1  -99
    1    0
    1  -99
    1    0
    1  -99
    7  -99
    1   23
    1   17
    1   25
    6  -99
    6  -99
    6  -99
    6  -99
    1  -99
    6  -99
    1  -99
    6  -99
    1  -99
    1  -99
    9  -99

