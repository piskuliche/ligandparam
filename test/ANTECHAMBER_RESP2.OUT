
 -----------------------------------------------
      Restrained ESP Fit 2.3  Amber 4.1
 -----------------------------------------------
 Resp charges for organic molecule                                               
 -----------------------------------------------


 inopt       =     0   ioutopt     =     1
 nmol        =     1   iqopt       =     2
 ihfree      =     1   irstrnt     =     1
 iunits      =     0   qwt         =   0.00100000
 wtmol(1) =     1.000000
 subtitle:

Resp charges for organic molecule                                               
ich =   0  iuniq =  41
    1    6    0
    2    6    0
    3    1    0
    4    1    3
    5    1    0
    6    1    5
    7    6  -99
    8    8  -99
    9    7  -99
   10    1  -99
   11    6  -99
   12    6  -99
   13    6  -99
   14    6    0
   15    6  -99
   16    6  -99
   17    1    0
   18    6    0
   19    6  -99
   20    1  -99
   21    6    0
   22    1  -99
   23    1    0
   24    1  -99
   25    1    0
   26    1  -99
   27    7  -99
   28    1   23
   29    1   17
   30    1   25
   31    6  -99
   32    6  -99
   33    6  -99
   34    6  -99
   35    1  -99
   36    6  -99
   37    1  -99
   38    6  -99
   39    1  -99
   40    1  -99
   41    9  -99
 new q0 values to be read  41

  ----------------------------------------------------------------------------
     ATOM                       COORDINATES                         CHARGE
                       X             Y                Z
  ----------------------------------------------------------------------------
  ----------------------------------------------------------------------------


 Charge on the molecule(ich) =    0
 Total number of atoms (iuniq)     =   41
 Weight factor on initial charge restraints(qwt)=     0.10000D-02


  there are  1 charge constraints:


     1       1
     2       1
     3       1
     4       1
     5       1
     6       1
     7       1
     8       1
     9       1
    10       1
    11       1
    12       1
    13       1
    14       1
    15       1
    16       1
    17       1
    18       1
    19       1
    20       1
    21       1
    22       1
    23       1
    24       1
    25       1
    26       1
    27       1
    28       1
    29       1
    30       1
    31       1
    32       1
    33       1
    34       1
    35       1
    36       1
    37       1
    38       1
    39       1
    40       1
    41       1

 Reading esp"s for molecule   1
 total number of atoms      =    41
 total number of esp points =  1634


 center     X       Y       Z 
    1  -0.1756850E+01   0.9834060E+01   0.2047102E+00
    2  -0.3163288E+01   0.1158121E+02  -0.1618836E+01
    3  -0.1824871E+00   0.8932483E+01  -0.7953233E+00
    4  -0.9166815E+00   0.1100202E+02   0.1690273E+01
    5  -0.3918315E+01   0.1049944E+02  -0.3219749E+01
    6  -0.4803315E+01   0.1240972E+02  -0.6503397E+00
    7  -0.1491423E+01   0.1374523E+02  -0.2513540E+01
    8   0.2078566E+00   0.1459274E+02  -0.1205740E+01
    9  -0.2114509E+01   0.1464625E+02  -0.4856428E+01
   10  -0.3557220E+01   0.1376108E+02  -0.5737166E+01
   11   0.9031189E+00   0.1817097E+02  -0.5284683E+01
   12   0.1837898E+01   0.2017522E+02  -0.6721884E+01
   13  -0.1073017E+01   0.1668660E+02  -0.6199938E+01
   14   0.4027745E+01   0.2169307E+02  -0.5697244E+01
   15   0.8223766E+00   0.2072272E+02  -0.9100571E+01
   16  -0.2100359E+01   0.1722819E+02  -0.8577629E+01
   17   0.5749138E+01   0.2050397E+02  -0.5831810E+01
   18   0.1823843E+01   0.2289821E+02  -0.1064602E+02
   19  -0.1154281E+01   0.1921668E+02  -0.9991350E+01
   20  -0.3640436E+01   0.1608269E+02  -0.9318398E+01
   21   0.4423635E+01   0.2372023E+02  -0.9728799E+01
   22   0.5872066E+01   0.2497955E+02  -0.6384360E+01
   23   0.1899156E+01   0.2237577E+02  -0.1265165E+02
   24  -0.1966511E+01   0.1961195E+02  -0.1183865E+02
   25   0.4960654E+01   0.2550832E+02  -0.1062236E+02
   26   0.1697475E+01   0.1774939E+02  -0.3445645E+01
   27   0.4294238E+01   0.2408703E+02  -0.7010761E+01
   28   0.5294937E+00   0.2451291E+02  -0.1048634E+02
   29   0.3723114E+01   0.2206679E+02  -0.3681827E+01
   30   0.5846126E+01   0.2228863E+02  -0.1028796E+02
   31  -0.3471400E+01   0.7856630E+01   0.1321342E+01
   32  -0.3763922E+01   0.5515619E+01   0.1385793E+00
   33  -0.4855335E+01   0.8351888E+01   0.3517587E+01
   34  -0.5407279E+01   0.3747948E+01   0.1157924E+01
   35  -0.2714745E+01   0.5041891E+01  -0.1561798E+01
   36  -0.6495994E+01   0.6550711E+01   0.4508000E+01
   37  -0.4630078E+01   0.1016175E+02   0.4464444E+01
   38  -0.6792244E+01   0.4216642E+01   0.3331723E+01
   39  -0.7550474E+01   0.6959096E+01   0.6221864E+01
   40  -0.8050877E+01   0.2775386E+01   0.4068342E+01
   41  -0.5658329E+01   0.1507064E+01  -0.5706972E-03
 Initial ssvpot =     0.356


 Number of unique UNfrozen centers=   10

 Non-linear optimization requested.
 qchnge =    0.3451893181E-01
 qchnge =    0.1404188788E-02
 qchnge =    0.2907350256E-03
 qchnge =    0.8230919618E-04
 qchnge =    0.2441238034E-04
 qchnge =    0.7294917401E-05
 qchnge =    0.2183068498E-05
 qchnge =    0.6535215570E-06

 Convergence in    7 iterations


Resp charges for organic molecule                                               

          Point Charges Before & After Optimization

    no.  At.no.    q(init)       q(opt)     ivary    d(rstr)/dq
    1   6      -0.015115      -0.007257      0       0.009974
    2   6      -0.091160      -0.082902      0       0.007698
    3   1       0.044979       0.041807      0       0.000000
    4   1       0.047339       0.041807      3       0.000000
    5   1       0.030288       0.038800      0       0.000000
    6   1       0.054399       0.038800      5       0.000000
    7   6       0.552116       0.552116    -99       0.001782
    8   8      -0.532669      -0.532669    -99       0.001845
    9   7      -0.360834      -0.360834    -99       0.002671
   10   1       0.269147       0.269147    -99       0.000000
   11   6      -0.101842      -0.101842    -99       0.007006
   12   6      -0.013966      -0.013966    -99       0.009904
   13   6       0.058661       0.058661    -99       0.008625
   14   6       0.049465       0.116363      0       0.006517
   15   6       0.099856       0.099856    -99       0.007076
   16   6      -0.240888      -0.240888    -99       0.003834
   17   1       0.046548       0.042427      0       0.000000
   18   6      -0.049826      -0.145554      0       0.005662
   19   6      -0.217365      -0.217365    -99       0.004179
   20   1       0.167072       0.167072    -99       0.000000
   21   6       0.027264       0.110641      0       0.006706
   22   1       0.358375       0.358375    -99       0.000000
   23   1       0.025344       0.072960      0       0.000000
   24   1       0.150393       0.150393    -99       0.000000
   25   1       0.066846       0.027135      0       0.000000
   26   1       0.126583       0.126583    -99       0.000000
   27   7      -0.693425      -0.693425    -99       0.001427
   28   1       0.062584       0.072960     23       0.000000
   29   1       0.088834       0.042427     17       0.000000
   30   1       0.049761       0.027135     25       0.000000
   31   6      -0.010631      -0.010631    -99       0.009944
   32   6      -0.177244      -0.177244    -99       0.004914
   33   6      -0.149802      -0.149802    -99       0.005552
   34   6       0.203433       0.203433    -99       0.004411
   35   1       0.151726       0.151726    -99       0.000000
   36   6      -0.228567      -0.228567    -99       0.004008
   37   1       0.153126       0.153126    -99       0.000000
   38   6      -0.123636      -0.123636    -99       0.006289
   39   1       0.171708       0.171708    -99       0.000000
   40   1       0.144082       0.144082    -99       0.000000
   41   9      -0.192958      -0.192958    -99       0.004601

 Sum over the calculated charges:     -0.000

        Statistics of the fitting:
  The initial sum of squares (ssvpot)                      0.356
  The residual sum of squares (chipot)                     0.007
  The std err of estimate (sqrt(chipot/N))               0.00200
  ESP relative RMS (SQRT(chipot/ssvpot))                 0.13536

 Center of Mass (Angst.):

 X  =   0.00000      Y  =   0.00000      Z  =   0.00000

 Dipole (Debye):

 X  =  -1.03363      Y  =   0.11056      Z  =  -1.62578

 Dipole Moment (Debye)=   1.92971

 Quadrupole (Debye*Angst.):

 Qxx =  19.23601     QYY = -57.09791     QZZ =  37.86189
 Qxy = -40.71235     QXZ = -49.55953     QYZ = -86.42140
